src/providers/node/mod.rs

```diff
@@ -580,16 +570,34 @@ fn version_number_to_pkg(version: u32) -> String {
     }
 }
 
-fn parse_regex_into_pkg(re: &Regex, node_version: &str) -> Option<String> {
-    let matches: Vec<_> = re.captures_iter(node_version).collect();
-    if let Some(captures) = matches.get(0) {
-        match captures[1].parse::<u32>() {
-            Ok(version) => return Some(version_number_to_pkg(version)),
-            Err(_e) => {}
+fn pkg_to_version_number(pkg: &str) -> Option<u32> {
+    let re = Regex::new(r"nodejs-(?P<version>\d+)_x").unwrap();
+    match re.captures(pkg) {
+        None => None,
+        Some(captures) => {
+            let version_number: u32 = captures["version"].parse().unwrap();
+            Some(version_number)
         }
     }
+}
```

Comment:
This function only really exists to parse `DEFAULT_NODE_PKG_NAME`. Instead, I think we should have the default node version be a number and then simply call `version_number_to_pkg` on it when we need to return a package string.

Reply:
ok done